backend/app/__init__.py

```diff
@@ -12,6 +25,164 @@ def home():
 @app.route("/health")
 def health():
     return "Works"
+  
+@app.route("/login")
+def login():
+    auth_url = createSpotifyOAuth().get_authorize_url()
+    return redirect(auth_url)
+
+@app.route("/authorize")
+def authorize():
+    return 'authorize'
+
+def createSpotifyOAuth():
+    return SpotifyOAuth(
+        client_id=''.join(CLIENT_ID),
+        client_secret=''.join(CLIENT_SECRET),
+        redirect_uri=url_for('authorize', _external=True),
+        scope='playlist-modify-public user-library-read user-library-modify user-read-email user-read-private',
+    )
+    
+@app.route("/imageToEmotion", methods=['POST'])
+def imageToEmotion():
+    """
+    Detect emotion in image
+    ___ 
+    post:
+        form-data:
+        - image : jpg file
+
+        response:
+            200:
+                content: [emotion, score]
+            500 if model fails to detect / interval server errore 
+
+
+    """
+    try:
+        image = request.files.get('image')
+        result = getEmotion(image)
+        if result:
+            return jsonify(result), 200
+        else:
+            return "No result available", 500
+    except Exception as e:
+            return str(e), 500
+
+
+#The following code is done with the help of this link:
+# https://www.youtube.com/watch?v=xdq6Gz33khQ
```

Comment:
Comments in form of docstrings are the ideal. Good thing to look at.

Reply:
Done 🚀